### PR TITLE
[WIP][upb] μpb bump to Aug 2020 code base

### PR DESCRIPTION
**Pull request by jozefizso**
**Describe the pull request**
- What does your PR fix? **Use latest μpb code from Aug 2020.**
- Which triplets are supported/not supported? I think all triplets are supported but ARM & UWP.
- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)? Yes.

### CODE REVIEW

File: ports/upb/CONTROL [hunk 1]
```diff
@@ -1,4 +1,4 @@
 Source: upb
-Version: 2019-10-21-2
+Version: 2020-08-04
```
Comment (NancyLi1013) [inline ports/upb/CONTROL:2]:
```suggestion
Version: 2020-08-19
```
The version should be a date that you accept the commit.
Please see https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/control-files.md#version

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

File: ports/upb/CONTROL (hunk 1 above)
Comment (jozefizso) [reply ports/upb/CONTROL]: Fixed.

**Review by NancyLi1013 — APPROVED** [review]

**Comment by BillyONeal** [comment]
Hi there @jozefizo ; the build failures now look legitimate rather than the infrastructure-caused 'soil' thing. Can you take a look?

**Comment by jozefizso** [comment]
@BillyONeal I think this upgrade of upb breaks existing grpc... I will need to take closer look at it.

**Comment by jozefizso** [comment]
The latest upb works for me only in grpc 1.31 (and the latest grpc does not work with existing upb in ports).

So I will integrate this into next grpc update.

**Comment by NancyLi1013** [comment]
Hi @jozefizso 
There is a PR #12920 to update `grpc `to 1.30.2. Does this version work for you?

**Comment by jozefizso** [comment]
No, the 1.30.x requires old upb.

The next 1.31.x was changed to latest upb.

**Pull request closed without merging**